java/android/nnstreamer/src/main/jni/nnstreamer-native-api.c

```diff
@@ -205,7 +212,12 @@ nns_construct_pipe_info (JNIEnv * env, jobject thiz, gpointer handle,
   g_mutex_init (&pipe_info->lock);
 
   (*env)->GetJavaVM (env, &pipe_info->jvm);
-  g_assert (pipe_info->jvm);
+  if (!pipe_info->jvm) {
+    _ml_loge ("Failed to get Java VM.");
+    nns_destroy_pipe_info (pipe_info, env);
```

Comment:
You called nns_destroy_pipe_info() before creating all params (e.g., pipe_info->cls tensors_data_cls_info tensors_info_cls_info).
This causes null-ptr exception.
I suggest just removing this line, handling all error case is too complex.